config/locales/gems/pwned/en.yml

```diff
@@ -0,0 +1,5 @@
+en:
+  errors:
+    messages:
+      not_pwned: you entered is insecure, please choose a stronger one
+      pwned_error: you entered might be insecure
```

Comment:
```suggestion
      pwned_error: your password choice might be insecure
```

Reply:
GEM varsayılan olarak 'password' prefix'i koyduğu için bu şekilde. Yani ekrana basılırken şöyle görünüyor:

'Password you entered might be insecure'